docs/content/release_notes/limitations.md

```diff
@@ -80,4 +80,10 @@ The following IBM block storage CSI driver features are not supported on volumes
 
 ## I/O group limitations
 
-**Note:** I/O group configuration is only supported for use with IBM Spectrum Virtualize family storage systems.
\ No newline at end of file
+**Note:** I/O group configuration is only supported for use with IBM Spectrum Virtualize family storage systems.
+
+## Automatic host definition limitations
```

Comment:
I suggest to change the "automatic" host definition to "dynamic" host definition, to be aligned with dynamic volume provisioning

Reply:
@erantzabari ? Your call

Reply:
per Slack from Eran - Dynamic host definition

Reply:
updated throughout branch the name of the feature

Reply:
I still see the word "automatic" in this diff, I think we should completely avoid it

Reply:
removed

Reply:
I still see:
>defining the host attachment is now automatic

which is also misleading, because `dynamicNodeLabeling` is not set by default.
we could write something like "defining hosts connectivity on storage can now be dynamic, if ..."

Reply:
there's no more "automatic" in regards to dynamic host define. please recheck